src/js/ui/CheckButton/index.jsx

```diff
@@ -13,12 +13,12 @@ const CheckButton = ({children, active, onClick, size, align, isCheckBox}) => (
     large: size === 'l',
     small: size === 's'
   })} size="lg" color="link" block active={active} onClick={onClick}>
-    {children}
     {!isCheckBox && <FontAwesomeIcon className={classnames({"float-right": align === "right", "float-left": align !== "right", "text-primary": active, "text-secondary": !active})}
                      icon={active ? faCircleReg : faCircle}/>}
     {isCheckBox && <span className={classnames("d-inline-block check-box text-center", {'bg-secondary': !active, 'bg-primary': active, "float-right": align === "right", "float-left": align !== "right"})}>
       {active && <FontAwesomeIcon className="text-white" icon={faCheck}/>}
     </span>}
+    {children}
```

Comment:
Is the text now to the right?

Reply:
The checkbox/radio have a float that will position it before the text.